transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/specification/extensions/idletimeout/IdleTimeoutExtensionWSNAcceptorIT.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2007-2014 Kaazing Corporation. All rights reserved.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.kaazing.gateway.transport.wsn.specification.extensions.idletimeout;
+
+import static org.kaazing.test.util.ITUtil.createRuleChain;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.kaazing.gateway.transport.IoHandlerAdapter;
+import org.kaazing.gateway.transport.wsn.WsnAcceptorRule;
+import org.kaazing.k3po.junit.annotation.Specification;
+import org.kaazing.k3po.junit.rules.K3poRule;
+import org.kaazing.mina.core.session.IoSessionEx;
+
+/**
+ * Class performing x-kaazing-idle-timeout-extension WSN acceptor robot tests integrated with the gateway
+ *
+ */
+public class IdleTimeoutExtensionWSNAcceptorIT {
+
+    private final K3poRule k3po = new K3poRule().setScriptRoot("org/kaazing/specification/ws.extensions/x-kaazing-idle-timeout");
+
+    private final WsnAcceptorRule acceptor = new WsnAcceptorRule();
+
+    @Rule
+    public TestRule chain = createRuleChain(acceptor, k3po);
+
+    @Test
+    @Specification("standard.pong.frames.sent.by.server.no.client.timeout/request")
+    public void standardPongFramesSentByServerNoClientTimeout() throws Exception {
+        acceptor.bind("ws://localhost:8001/echo", new IoHandlerAdapter<IoSessionEx>());
```

Comment:
You'll need to make the handler send data every tick seconds (as in the corresponding response.rpt script). You can override method "sessionOpened" to do that, and use session.write to write the data. You'll need to create an IoBufferEx object to contain the data, using the buffer allocator you can get from the session.